modules/Microsoft.Web/staticSites/readme.md

```diff
@@ -269,6 +269,7 @@ userAssignedIdentities: {
 | `resourceGroupName` | string | The resource group the static site was deployed into. |
 | `resourceId` | string | The resource ID of the static site. |
 | `systemAssignedPrincipalId` | string | The principal ID of the system assigned identity. |
+| `defaultHostname` | string | The default autogenerated hostname for the static site. |
```

Comment:
Please see other comment as there is a similar error

![image](https://user-images.githubusercontent.com/28486158/185042968-438451a6-1ba8-4bad-ae53-a34e3c4f857f.png)